test/pass/licenseTest.scss

```diff
@@ -0,0 +1,27 @@
+/*
+ * backpack-node-sass
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
Do we need to regenerate this test? The comment block at the top is now different from what's inside `test/pass/license.txt`

Reply:
Nah as this is just a file included it actually makes no difference if there is a comment in there or not on this test. This file was manually created 😆

Reply:
I've actually just deleted it as it makes no difference to the tests as the `test.scss` will cover this anyway